Formula/dum.rb

```diff
@@ -0,0 +1,25 @@
+class Dum < Formula
+  desc "Npm scripts runner written in Rust"
+  homepage "https://github.com/egoist/dum"
+  url "https://github.com/egoist/dum/archive/v0.1.8.tar.gz"
+  sha256 "0c8337d79e0e331b2dbd14e615adc9d359d3491632e21d479a306c9a2845009c"
+  license "MIT"
+  head "https://github.com/egoist/dum.git", branch: "master"
+
+  livecheck do
+    url :stable
+    regex(/^v?(\d+(?:\.\d+)+)$/i)
+  end
+
+  depends_on "rust" => :build
+
+  uses_from_macos "zlib"
+
+  def install
+    system "cargo", "install", *std_cargo_args
+  end
+
+  test do
+    system "false"
```

Comment:
This needs a proper test.

Reply:
I am finding a test suit to make sure to install it successfully. Should be added tomorrow.

Reply:
Please don't try and run an upstream test suite, those are not the types of tests we're looking for here. See https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

Reply:
> Please don't try and run an upstream test suite, those are not the types of tests we're looking for here. See https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

Thanks, I will finish in right way👌